### PR TITLE
Geomedian Prod OWS and DEA Maps

### PR DESCRIPTION
**Pull request by JM-GA**
Deploy new version of geomedian OWS and DEA Maps in production.
Note: Previous version will be available for a transitional time, before being removed in a separate PR

### CODE REVIEW

File: prod/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py [hunk 1]
```diff
@@ -8,6 +10,153 @@
...
+
+Euclidean distance (EMAD)
+Cosine (spectral) distance (SMAD)
+Bray Curtis dissimilarity (BCMAD)
+Together, they provide information on variance in the landscape over the given year and are useful for change detection applications.
+
+v4.0.0 Changelog:
+Cloud masks have been cleaned using a 3-pixel morphological opening on clouds and a 6-pixel dilation on cloud and shadows.
+The south-west origin point of the DEA Summary Product Grid has been shifted 18 tiles west and 15 tiles south in this version.
+A single combined product for Landsat 8 and 9 is provided. A standalone Landsat 8 product will no longer be provided from calendar year 2022 and o$
```
Comment (robbibt) [inline prod/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py:109]: Is the `o$' here intentional?

Comment (JM-GA) [reply prod/services/wms/ows_refactored/baseline_satellite_data/landsat_annual/ows_landsat_annual_cfg.py]: good pickup! have corrected

**Review by robbibt — APPROVED** [review]
This looks great to me - will do a more detailed test of DEA Maps and the web services themselves when they are live

**Comment by JM-GA** [comment]
@SpacemanPaul or @matt-fry-ga  - any issues in OWS prod deployments you can see happening - guessing the issues we had in dev before are now resolved? wondering if I should test deploying now or wait until Monday - thanks!

**Comment by matt-fry-ga** [comment]
As a general rule I don't release things on Friday afternoon unless it's to fix something that's broken.  Can we wait until Monday please?

**Comment by JM-GA** [comment]
@matt-fry-ga  - yep happy to wait until Monday to deploy

**Pull request merged**